client/java/src/main/java/io/openlineage/client/transports/HttpConfig.java

```diff
@@ -28,7 +29,29 @@ public final class HttpConfig implements TransportConfig {
       timeout; // deprecated, will be removed in 1.13, assumes timeout is in seconds
 
   @Getter @Setter private @Nullable Integer timeoutInMillis;
-  @Getter @Setter private @Nullable TokenProvider auth;
-  @Getter @Setter private @Nullable Map<String, String> urlParams;
-  @Getter @Setter private @Nullable Map<String, String> headers;
+
+  @JsonProperty(access = JsonProperty.Access.WRITE_ONLY)
```

Comment:
added to prevent serialising the field within debug facet

Reply:
👍